.github/workflows/update-deps.yml

```diff
@@ -43,3 +43,38 @@ jobs:
           labels: ok-to-test
           body: |
             Updating go.mod with latest dependencies...
+  update-go-version:
```

Comment:
Does this need to be a separate job?

Reply:
maybe not, but separate out to avoid one failure affect the other?